Assets/MRTK/SDK/Features/UX/Scripts/Dwell/DwellProfile.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Dwell
+{
+    /// <summary>
+    /// Profile used by dwell handler to configure the various thresholds.
+    /// </summary>
+    [MixedRealityServiceProfile(typeof(DwellProfile))]
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Dwell Profile", fileName = "DwellProfile", order = 100)]
+    [Serializable]
+    public class DwellProfile : BaseMixedRealityProfile
```

Comment:
I don't think this should derive from BaseMixedRealityProfile, as that's used exclusively by the MixedReality Profile system at the moment (all the current derivatives also have a naming pattern like MixedRealityRegisteredServiceProvidersProfile too)

I have some concerns with calling this a "profile", since I though that would be used exclusively by the main MRTK object, but I see that URP also uses "profiles" for some of it's components, so it's fine with me.

Reply:
I agree that the use of the word "profile" should be more consistent. However as we have already been using this name I would suggest that we take the opportunity to rename in 3.0. The base class has been changed to ScriptableObject.